src/web/core/src/main/java/org/geoserver/web/data/resource/ResourceDimensionsTabPanelInfo.java

```diff
@@ -58,7 +58,7 @@ public ResourceDimensionsTabPanelInfo(String id, IModel<LayerInfo> model) {
         if (time.getObject() == null) {
             time.setObject(new DimensionInfoImpl());
         }
-        add(new DimensionEditor("time", time, resource, Date.class));
+        add(new DimensionEditor("time", time, resource, Date.class, true));

```

Comment:
So by default nearest match is not activate and is only activate for time dimensions.

Reply:
Correct, see "backwards compatibility" in the proposal (and also the proposal for only supporting time in this round of coding)